client/src/components/CheckoutForm/CheckoutForm.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Formik, Form } from 'formik';
+import { Stepper, Step, StepLabel, Button } from '@mui/material';
+import { UserDetailsForm, ShippingAddressForm, PaymentForm, PaymentSuccess, CheckoutSummary } from '..';
+import checkoutFormModel from '../../utils/checkoutFormModels/checkoutFormModel';
+import CheckoutFormInitialValues from '../../utils/checkoutFormModels/CheckoutFormInitialValues';
+import CheckoutFormvalidationSchema from '../../utils/checkoutFormModels/CheckoutFormvalidationSchema';
```

Comment:
```suggestion
import CheckoutFormValidationSchema from '../../utils/checkoutFormModels/CheckoutFormValidationSchema';
```

Reply:
filename is validationSchema with a lowercase v,   this is a temporary naming scheme since it will be renamed after being moved to the file structure.